prep-javascript-objects/main.js

```diff
@@ -0,0 +1,19 @@
+const person = {
+  firstName: 'Aaron',
+  lastName: 'He',
+  hobby: 'Sleeping',
+  job: 'Frac Operator'
+};
+console.log(person);
+
+const fullName = "The person's name is: " + person.firstName + " " + person.lastName;
+console.log(fullName);
+
+const job = "The person's current job is " + person.job;
```

Comment:
the job property should be created using dot notation. It should not be declared in the initial object